code/game/turfs/turf.dm

```diff
@@ -186,11 +187,14 @@
 			lighting_build_overlays()
 		else
 			lighting_clear_overlays()
-	
+
 	obscured = old_obscured
 
 	return W
 
+/turf/proc/PreChange()
+	return
+
```

Comment:
`Pre`, `Before`, they mean the same thing in this context, but `Before is probably a better antonym for `After` than `Pre` is.

And yeah, I'll move `ChangeTurf` overrides to this proc now, might as well